package.json

```diff
@@ -1,21 +1,34 @@
 {
-	"name": "country-select-js",
-	"version": "1.0.9",
-	"description": "A jQuery plugin for selecting a country",
-	"main": "build/js/countrySelect.js",
-	"repository": {
-		"type": "git",
-		"url": "https://mrmarkfrench@github.com/mrmarkfrench/country-select-js.git"
-	},
-	"keywords": [
-		"country",
-		"international",
-		"jQuery"
-	],
-	"author": "Mark French",
-	"license": "MIT",
-	"bugs": {
-		"url": "https://github.com/mrmarkfrench/country-select-js/issues"
-	},
-	"homepage": "https://github.com/mrmarkfrench/country-select-js"
+  "name": "country-select-js",
```

Comment:
My bad, it's been a while since i don't code on sources with `tab` indentation, so my editor was programmed to convert it autommaticaly. Sorry